app/controllers/schools_controller.rb

```diff
@@ -44,7 +44,9 @@ def school_administrator_dashboard
   #Individual dashboard data endpoints will replace school_administrator_dashboard once they're stable
   def absence_dashboard_data
     student_absence_data = students_for_dashboard(@school)
-      .includes([homeroom: :educator], :dashboard_absences, :event_notes)
+      .includes([homeroom: :educator], :absences, :event_notes)
+      .where("absences.occurred_at >= ?", 1.year.ago).references(:absences)
```

Comment:
When I look at the absence dashboard on this branch, it looks like this: 
![screen shot 2018-04-16 at 5 49 55 pm](https://user-images.githubusercontent.com/638809/38837204-147ba0c8-419f-11e8-8870-5ac77d1662f8.png)

But on master, it looks like this: 
![screen shot 2018-04-16 at 5 50 09 pm](https://user-images.githubusercontent.com/638809/38837221-228c7462-419f-11e8-87de-4daf501e9a92.png)

Note the percentages are significantly higher. I think adding the where clause here applies the where filter to the entire includes query, not just the absences. That filters out all students with no absences within the past year.

Reply:
@edavidsonsawyer Ooooh, good catch here!